demo-app/client/src/tests/patch/patch-order.service.spec.ts

```diff
@@ -0,0 +1,44 @@
+// TODO: make the test case work
```

Comment:
The test was not passing. Will be fixed in the next PR. It is connected to how bodyParams are created.